lib/css/media.css

```diff
@@ -1,6 +1,12 @@
 .tsf-image-notifications {
 	line-height: 1;
 	vertical-align: bottom;
+
+	> .tsf-tooltip-wrap {
+		/* In gutenberg, this ensures the wrap has the size of its contents, not sure why it’s
+		   not required in the classic editor. */
+		display: inline-block;
```

Comment:
I cannot reproduce this error on Chromium. What browser are you using?

Reply:
Firefox, Safari and Chrome all show me the same thing. Probably, it’s just not clear what to look for. Here’s a recording that should clarify:

https://github.com/user-attachments/assets/ccf7b896-518e-496b-a85d-1d64bf827fa6

Apparently the popover component in Gutenberg reads the wrap’s rect differently. So, of course, this isn’t required in regular admin screens - the comment could have been better. I suppose this style should move to gbc.css and maybe applied generically to all wraps if it doesn’t mess up any other wraps.